### PR TITLE
MM-21561 add mmctl subcommands `command modify | show | move`

### PR DESCRIPTION
**Pull request by wiggin77**
#### Summary
This PR adds mmctl subcommands:
- command modify
- command show
- command move

Blocking on PR's https://github.com/mattermost/mattermost-server/pull/13624 and https://github.com/mattermost/mattermost-server/pull/13598

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-21561

### CODE REVIEW

File: commands/command_test.go [hunk 1]
```diff
@@ -469,3 +472,493 @@ func (s *MmctlUnitTestSuite) TestCommandListCmdF() {
 		s.Equal("Unable to list commands for '"+teamID+"'", printer.GetErrorLines()[0])
 	})
 }
+
+func (s *MmctlUnitTestSuite) helperCreateCommand(*model.Command) {
+	s.T().Helper()
+
```
Comment (mgdelacroix) [inline commands/command_test.go]: Unnecessary whitespace

File: commands/command_test.go [hunk 2]
```diff
@@ -469,3 +472,493 @@ func (s *MmctlUnitTestSuite) TestCommandListCmdF() {
 		s.Equal("Unable to list commands for '"+teamID+"'", printer.GetErrorLines()[0])
 	})
 }
+
+func (s *MmctlUnitTestSuite) helperCreateCommand(*model.Command) {
```
Comment (mgdelacroix) [inline commands/command_test.go]: What is this used for?

File: commands/command.go [hunk 3]
```diff
@@ -33,34 +34,72 @@ var CommandListCmd = &cobra.Command{
...
+var CommandShowCmd = &cobra.Command{
+	Use:     "show [commandID]",
+	Short:   "Show a custom slash command",
+	Long:    `Show a custom slash command. Commands can be specified by command ID. Returns command ID, team ID, trigger word, display name and creator username.`,
+	Args:    cobra.MinimumNArgs(1),
+	Example: `  command show commandID`,
+	RunE:    withClient(showCommandCmdF),
+}
+
+func addFlags(cmd *cobra.Command) {
```
[the remark this answers is not in the document]
Comment (wiggin77) [reply commands/command.go]: Addressed via https://github.com/mattermost/mmctl/pull/103/commits/1c6d7a61e9deaa09a2226de724b92757b116a7dc

File: commands/command.go [hunk 4]
```diff
@@ -33,34 +34,72 @@ var CommandListCmd = &cobra.Command{
...
+	Args:    cobra.MinimumNArgs(1),
+	Example: `  command modify commandID --title MyModifiedCommand --description "My Modified Command Description" --trigger-word mycommand --url http://localhost:8000/my-slash-handler --creator myusername --response-username my-bot-username --icon http://localhost:8000/my-slash-handler-bot-icon.png --autocomplete --post`,
+	RunE:    withClient(modifyCommandCmdF),
+}
+
+var CommandMoveCmd = &cobra.Command{
+	Use:     "move [team] [commandID]",
+	Short:   "Move a slash command to a different team",
+	Long:    `Move a slash command to a different team. Commands can be specified by command ID.`,
+	Args:    cobra.MinimumNArgs(2),
```
[the remark this answers is not in the document]
Comment (wiggin77) [reply commands/command.go]: Addressed via https://github.com/mattermost/mmctl/pull/103/commits/1c6d7a61e9deaa09a2226de724b92757b116a7dc

File: commands/command.go [hunk 5]
```diff
@@ -33,34 +34,72 @@ var CommandListCmd = &cobra.Command{
...
 	Example: `  command delete commandID`,
 	Args:    cobra.ExactArgs(1),
 	RunE:    withClient(deleteCommandCmdF),
 }
 
+var CommandModifyCmd = &cobra.Command{
+	Use:     "modify [commandID]",
+	Short:   "Modify a slash command",
+	Long:    `Modify a slash command. Commands can be specified by command ID.`,
+	Args:    cobra.MinimumNArgs(1),
```
[the remark this answers is not in the document]
Comment (wiggin77) [reply commands/command.go]: Thanks.  I've not use Cobra, and I need to brush up.  Addressed via https://github.com/mattermost/mmctl/pull/103/commits/d59b21e49e373429e4f8fb38edf631b52bffd2c8

File: commands/command.go [hunk 6]
```diff
@@ -33,34 +34,72 @@ var CommandListCmd = &cobra.Command{
...
+	Args:    cobra.MinimumNArgs(2),
+	Example: `  command move newteam commandID`,
+	RunE:    withClient(moveCommandCmdF),
+}
+
+var CommandShowCmd = &cobra.Command{
+	Use:     "show [commandID]",
+	Short:   "Show a custom slash command",
+	Long:    `Show a custom slash command. Commands can be specified by command ID. Returns command ID, team ID, trigger word, display name and creator username.`,
+	Args:    cobra.MinimumNArgs(1),
```
[the remark this answers is not in the document]
Comment (wiggin77) [reply commands/command.go]: Addressed via https://github.com/mattermost/mmctl/pull/103/commits/2606a3bd7fb0a56db77b18956389a9cfe7c8eb96

File: commands/command_test.go (hunk 1 above)
Comment (wiggin77) [reply commands/command_test.go]: Thanks @mgdelacroix   This was left over from some experimental unit test helper that I missed when cleaning up.  Removed.

File: commands/command_test.go (hunk 2 above)
Comment (wiggin77) [reply commands/command_test.go]: Removed per above.
